<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widgetDigit" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSave">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTrain">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
    <property name="text">
     <string>TRAIN MODEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPredict">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
    <property name="text">
     <string>PREDICT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPredict1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPredict2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pbClear</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbTrain</tabstop>
  <tabstop>pbPredict</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
